exercises/isogram/test/test_isogram.c

```diff
@@ -2,48 +2,61 @@
 #include "../src/isogram.h"
 #include <stdlib.h>
 
+#ifdef RUN_ALL_TESTS
+#undef TEST_IGNORE_MESSAGE
+#define TEST_IGNORE_MESSAGE(X)
+#endif
+
 void test_empty_string(void)
 {
    TEST_ASSERT_TRUE(isIsogram(""));
 }
 
 void test_lower_case_only(void)
 {
+   TEST_IGNORE_MESSAGE("Delete this line after the above test passes");
```

Comment:
I'd prefer this to be `TEST_IGNORE() // Delete this line to run the test`, and maybe only need to have the comment on the first one in each test suite. The message makes the first run output very busy:
```
Compiling tests.out
test/test_isogram.c:67:test_empty_string:PASS
test/test_isogram.c:17:test_lower_case_only:IGNORE: Delete this line after the above test passes
test/test_isogram.c:23:test_duplicated_letter:IGNORE: Delete this line after the above test passes
test/test_isogram.c:29:test_longest_known_isogram:IGNORE: Delete this line after the above test passes
test/test_isogram.c:35:test_duplicated_letter_mixed_case:IGNORE: Delete this line after the above test passes
test/test_isogram.c:41:test_non_letter_char:IGNORE: Delete this line after the above test passes
test/test_isogram.c:47:test_duplicated_non_letter_char:IGNORE: Delete this line after the above test passes
test/test_isogram.c:53:test_multiple_whitespace:IGNORE: Delete this line after the above test passes
test/test_isogram.c:59:test_duplicated_letter_within_word:IGNORE: Delete this line after the above test passes

-----------------------
9 Tests 0 Failures 8 Ignored
OK
```